carina-core/src/main/java/com/qaprosoft/carina/core/foundation/report/email/EmailReportGenerator.java

```diff
@@ -97,7 +99,6 @@
     private int skipCount = 0;
 
     public EmailReportGenerator(String title, String url, String version, String device, String browser, String finishDate, String elapsedTime,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 9 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://demo.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS00107)